pages/credits.vue

```diff
@@ -189,45 +182,37 @@
 	const totalDeductions = computed(() => credits.value.deductions.reduce((sum, deduction) => sum + deduction.amount, 0));
 	const dailyAdditions = computed(() => credits.value.todayOnlineProbes * creditsPerAdoptedProbe);
 
-	const loadLazyData = async () => {
-		loading.value = true;
-
-		try {
-			const { changes, count } = await $directus.request<{ changes: CreditsChange[]; count: number }>(customEndpoint({
-				method: 'GET',
-				path: '/credits-timeline',
-				params: {
-					userId: getUserFilter('user_id').user_id?._eq || 'all',
-					offset: first.value,
-					limit: itemsPerPage.value,
-				},
-			}));
-
-			creditsChanges.value = [
-				...changes.map(change => ({
-					...change,
-					date_created: formatDateForTable(change.date_created),
-				})),
-			];
-
-			creditsChangesCount.value = count;
-		} catch (e) {
-			sendErrorToast(e);
+	const { data: creditsData, pending: loading, error: creditsDataError } = await useLazyAsyncData(() => $directus.request<{ changes: CreditsChange[]; count: number }>(customEndpoint({
+		method: 'GET',
+		path: '/credits-timeline',
+		params: {
+			userId: getUserFilter('user_id').user_id?._eq || 'all',
+			offset: first.value,
+			limit: itemsPerPage.value,
+		},
+	})), { watch: [ page, itemsPerPage ] });
```

Comment:
```suggestion
	})), { watch: [ first, itemsPerPage ] });
```

Maybe this since that's what we actually use?